packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/AddAccountModal/AddAccountModal.tsx

```diff
@@ -27,6 +27,7 @@ import { DeviceModelInternal } from '@trezor/connect';
 
 const StyledModal = styled(Modal)`
     width: 480px;
+    padding-top: 4px;
     text-align: left;
```

Comment:
I secretly removed the coin symbol https://github.com/trezor/trezor-suite/pull/13734 😡

Reply:
🤬🤬🤬🤬🤬